journal-week-2.md

```diff
@@ -1,14 +1,14 @@
 
 ###Proof of Life
 
-We made it through the first week, and we are still alive. the second week was like a recovery period for us after the shock.  Most of us started to understand how this course will look like. That we will be bombarded with assignment especially on weekends, and from there we will learn to solve problems though research, team work and alot of educated guesses. 
+We made it through the first week, and we are still alive. the second week was like a recovery period for us after the shock. Most of us started to understand how this course will look like - that we will be bombarded with assignment especially on weekends, and from there we will learn to solve problems through research, team work and alot of educated guesses.
```

Comment:
You might want to capitalize the T in "The second week..." otherwise it's getting better.